openid-federation-1_0.xml

```diff
@@ -6532,7 +6532,7 @@ HTTP/1.1 302 Found
                     claim of the Entity Statement to the Trust Anchor it
                     selected to process the request. The
```

Comment:
With the changes above, the OP no longer selects a Trust Anchor, but a Trust Chain during request processing. Maybe rephrase to "... to the Trust Anchor of the Trust Chain it selected to process the request." to clarify.

Likewise below in the description of the `trust_anchor_id` registration response claim.